content/html/concepts/tags/terms/doctype/doctype.md

```diff
@@ -0,0 +1,36 @@
+---
+Title: '<!DOCTYPE>'
+Description: 'Tells the browser that the current file is in the HTML format.'
+Subjects:
+  - 'Web Development'
+  - 'Web Design'
+Tags:
+  - 'Browsers'
+  - 'HTML'
+CatalogContent:
+  - 'learn-html'
+  - 'paths/front-end-engineer-career-path'
+---
+
+The `<!DOCTYPE>` tag tells the browser that the current file is written in HTML and not another type such as JavaScript or CSS.
```

Comment:
```suggestion
The `<!DOCTYPE>` tag tells the browser that a given file is a markup (`.html`) file and that the browser must apply full web standards to it.
```

What's written in an HTML file isn't strictly limited to pure HTML. You can use a `<script>` tag to write custom JavaScript and a `<style>` tag in the `<head>` to apply CSS styles.